readthedocs/api/v3/views.py

```diff
@@ -110,14 +115,13 @@ class ProjectsViewSet(APIv3Settings, NestedViewSetMixin, ProjectQuerySetMixin,
     * Subprojects of a project: ``/api/v3/projects/{project_slug}/subprojects/``
     * Superproject of a project: ``/api/v3/projects/{project_slug}/superproject/``
 
-    Go to [https://docs.readthedocs.io/en/stable/api/v3.html](https://docs.readthedocs.io/en/stable/api/v3.html)
+    Go to [https://docs.readthedocs.io/page/api/v3.html](https://docs.readthedocs.io/page/api/v3.html)
```

Comment:
I think this should auto-link, no? Or you can put `<>` around it I believe (depending on markdown implementation, 😦)

Reply:
Yes. It's rendered as Markdown using "some implementation" and `[Text](Link)` is the format that works.